packages/node_modules/pouchdb-adapter-idb/src/index.js

```diff
@@ -387,26 +387,21 @@ function init(api, opts, callback) {
   };
 
   api._info = function idb_info(callback) {
-
-    if (idb === null || !cachedDBs.has(dbName)) {
-      var error = new Error('db isn\'t open');
-      error.id = 'idbNull';
-      return callback(error);
-    }
```

Comment:
Setting `idb` to `null` was not necessary; we can just check `cachedDBs`.

Also, there is no need to manually check for closed state because `adapter.js` already does that.